packages/components/addon/components/hds/code-block/index.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { assert } from '@ember/debug';
+import { htmlSafe } from '@ember/template';
+import Prism from 'prismjs';
+import { setup } from 'prismjs-glimmer';
```

Comment:
[question] is this dependency actually needed? Per [the docs](https://github.com/NullVoxPopuli/prismjs-glimmer#usage) this is specifically for highlighting glimmer/hbs type syntax which is not a use case we have in our consuming apps (that I'm aware of). If I comment this out I still see the syntax highlighting work correctly in our showcase.

Reply:
I suspect we don't – I'll try to get rid of it

Reply:
Removed it in 5f40c1db2662cea802b5563da89a8df4f1449d1a

Reply:
👏